### PR TITLE
Kevin Golden Hogwarts Exercise

### PR DESCRIPTION
**Pull request by kwg58**
Completeness 4/5
Comfort 3/5

Still getting comfortable with Rails.   One or two stupid little mistakes I haven't completely fixed, and the add student function is only halfway done, but I have to get out of here.

### CODE REVIEW

**Comment by kwg58** [comment]
I'm worried.  I just tried to run it and it didn't work.  When I tried to start the server I got "config.eager_load is set to nil. Please update your config/environments/*.rb files accordingly:" and now the application won't run.  I don't know what is going to happen when you open it....

**Comment by kwg58** [comment]
NEVER MIND.  It should work now.... stupid mistake on my part.

File: db/migrate/20131030184435_create_houses.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class CreateHouses < ActiveRecord::Migration
   def change
     create_table :houses do |t|
-      f.string :name, null: false
-      f.string :img_url
+      t.string :name, null: :false
```
Comment (amaseda) [inline db/migrate/20131030184435_create_houses.rb:4]: You don't need a colon before `false`. It's a boolean, not a symbol. You do still need it after `null` though.

**Comment by amaseda** [comment]
:+1: Good job with this. Let me know if you have any lingering questions.

**Pull request closed without merging**